sklearn/calibration.py

```diff
@@ -895,11 +901,18 @@ def calibration_curve(
 
         .. versionadded:: 1.1
 
-    normalize : bool, default=False
+    normalize : bool, default="deprecated"
         Whether y_prob needs to be normalized into the [0, 1] interval, i.e.
         is not a proper probability. If True, the smallest value in y_prob
         is linearly mapped onto 0 and the largest one onto 1.
 
+        .. deprecated:: 1.1
+            The normalize argument is deprecated in v1.1 and will be removed in v1.3.
+            Explicitly normalizing `y_prob` will reproduce this behavior, but it is
+            recommended that a proper probability is used (i.e. a classifier's
+            `predict_proba` positive class or `decision_function` output calibrated
+            with `CalibratedClassifierCV`).
```

Comment:
I see why `decision_function` is included here, but I think it can be confusing, because `CalibratedClassifierCV` itself does not have a `decision_function`. What do you think about removing the second part?

```suggestion
            `predict_proba` positive class).
```

Reply:
I think this is a fair point. What you suggest is more concise and less ambiguous.